Stripe/StripeiOSTests/ConsumerSessionTests.swift

```diff
@@ -175,9 +116,8 @@ class ConsumerSessionTests: XCTestCase {
             phoneNumber: "+13105551234",
             legalName: nil,
             countryCode: "US",
-            consentAction: nil,
-            with: apiClient,
-            cookieStore: cookieStore
+            consentAction: "clicked_checkbox_nospm_mobile_v0",
```

Comment:
use: `PaymentSheetLinkAccount.ConsentAction.checkbox_v0.rawValue` ?
though, this makes me think the function should accept an enum vs a string